libnavigation-base/build.gradle

```diff
@@ -4,15 +4,17 @@ apply plugin: 'org.jetbrains.dokka'
 apply plugin: 'com.jaredsburrows.license'
 apply from: "${rootDir}/gradle/ktlint.gradle"
 
-dokka {
-    outputDirectory = "$buildDir/kdoc"
-    configuration {
-        moduleName = 'libnavigation-base'
-        reportUndocumented = true
-
-        perPackageOption {
-            prefix = "com.mapbox.navigation.base.internal"
-            suppress = true
+dokkaHtml {
+    outputDirectory.set(new File(buildDir.path, "kdoc"))
+    moduleName.set("libnavigation-base")
```

Comment:
Let's use more public-facing names, like `Base`. `libnavigation-base` is just an internal name for a module. Same everywhere below.

Reply:
sounds good!

Reply:
Just a reminder about this :)